addon/modifiers/au-floating-ui.js

```diff
@@ -0,0 +1,158 @@
+import Modifier from 'ember-modifier';
+import { assert } from '@ember/debug';
+import { registerDestructor } from '@ember/destroyable';
+
+import {
+  autoUpdate,
+  computePosition,
+  flip,
+  hide,
+  offset,
+  arrow,
+} from '@floating-ui/dom';
+
+import { merge } from 'merge-anything';
```

Comment:
I needed some functionality to deep extend a default object. Is it ok to use external package like this?

Reply:
Sure, as long as it's listed as a `dependency` it should work in consuming apps as well.

Reply:
I don't have much experience regarding modifiers, but this seems like a correct conversion, right?

https://github.com/appuniversum/ember-appuniversum/pull/344/commits/0526e4b4c230acb88000c64f57361b273adf715c